### PR TITLE
New official website develop

### PR DESCRIPTION
**Pull request by duhenglucky**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/02-生产者/05message1.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Apache RocketMQ可用于以三种方式发送消息：**同步、异步和单向
 2. **设置 NameServer 的地址**。Apache RocketMQ很多方式设置NameServer地址(客户端配置中有介绍)，这里是在代码中调用producer的API setNamesrvAddr进行设置，如果有多个NameServer，中间以分号隔开，比如"127.0.0.2:9876;127.0.0.3:9876"。 
 3. **第三步是构建消息**。指定topic、tag、body等信息，tag可以理解成标签，对消息进行再归类，RocketMQ可以在消费端对tag进行过滤。
```
Comment (lizhanhui) [inline docs/02-生产者/05message1.md:62]:
```suggestion
3. **第三步是构建待发送消息**。构建消息时，可以指定topic、tag、body，以及自定义键值对等。其中tag可以对消息进行再打标，后续订阅消费时，可根据tag进行过滤。
```

File: docs/02-生产者/05message1.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Apache RocketMQ可用于以三种方式发送消息：**同步、异步和单向
...
 4. **最后调用send接口将消息发送出去**。同步发送等待结果最后返回SendResult，SendResut包含实际发送状态还包括SEND_OK（发送成功）, FLUSH_DISK_TIMEOUT（刷盘超时）, FLUSH_SLAVE_TIMEOUT（同步到备超时）, SLAVE_NOT_AVAILABLE（备不可用），如果发送失败会抛出异常。
```
Comment (lizhanhui) [inline docs/02-生产者/05message1.md:63]: 5.0， 返回的是SendReceipt. https://github.com/apache/rocketmq-clients/blob/master/java/client-apis/src/main/java/org/apache/rocketmq/client/apis/producer/SendReceipt.java

Comment (lizhanhui) [reply docs/02-生产者/05message1.md:63]: 5.0也不再包含存储层面的状态了

File: docusaurus.config.js [hunk 3]
```diff
@@ -124,20 +124,76 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
...
+              from: '/about/rewards/',
+              to: '/forth-blog'
+            },
             {
               from: '/docs/rmq-arc',
               to: '/docs/领域模型/01main'
             },
+            {
+              from: '/docs/rmq-deployment/',
+              to: '/docs/部署与运维/15deploy'
```
Comment (chenzlalvin) [inline docusaurus.config.js:160]: to:  '/docs/部署运维/15deploy'

File: docusaurus.config.js [hunk 4]
```diff
@@ -187,13 +187,17 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
               from: '/docs/cli-admin-tool/',
               to: '/docs/部署与运维/16admintool'
```
Comment (chenzlalvin) [inline docusaurus.config.js:188]: to: '/docs/部署运维/16admintool'

File: docusaurus.config.js [hunk 5]
```diff
@@ -153,7 +153,7 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
             },
             {
               from: '/docs/rmq-arc',
-              to: '/docs/介绍/03whatis'
+              to: '/docs/领域模型/01main'
             },
             {
               from: '/docs/rmq-deployment/',
```
Comment (chenzlalvin) [inline docusaurus.config.js:159]: to: '/docs/部署运维/15deploy'

**Pull request closed without merging**